doc/classes/ButtonGroup.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<class name="ButtonGroup" inherits="Resource" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+<class name="ButtonGroup" inherits="Resource" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd" keywords="radio">
 	<brief_description>
 		A group of buttons that doesn't allow more than one button to be pressed at a time.
 	</brief_description>
 	<description>
 		A group of [BaseButton]-derived buttons. The buttons in a [ButtonGroup] are treated like radio buttons: No more than one button can be pressed at a time. Some types of buttons (such as [CheckBox]) may have a special appearance in this state.
 		Every member of a [ButtonGroup] should have [member BaseButton.toggle_mode] set to [code]true[/code].
+		[b]Creating from the editor:[/b] To create a [ButtonGroup] with several buttons in one go, select all the [BaseButton]-derived nodes you want to group, then click the resource dropdown property for [b]Button Group[/b] in the inspector and choose [b]New ButtonGroup[/b]. This will create a single [ButtonGroup] instance shared across all selected buttons. To add more buttons to the [ButtonGroup] later, use the [b]Copy[/b] action on the [b]Button Group[/b] resource dropdown, then select another [BaseButton]-derived node and use the [b]Paste[/b] action on its [b]Button Group[/b] resource dropdown.
```

Comment:
It's so long I'm not sure who is going to read all of this. I suggest the following, but in my opinion it should be simplified further.
```suggestion
		[b]Creating from the editor:[/b] To create a [ButtonGroup] with several buttons at once, select all the [BaseButton]-inheriting nodes you want to group, then click the resource dropdown property for [b]Button Group[/b] in the Inspector and choose [b]ButtonGroup[/b]. This will create a single [ButtonGroup] instance shared across all selected buttons. To add more buttons to the [ButtonGroup] later, use the [b]Copy[/b] action on the [b]Button Group[/b] resource dropdown, then select another [BaseButton]-derived node and use the [b]Paste[/b] action on its [b]Button Group[/b] resource dropdown.
```